uvloop/includes/stdlib.pxi

```diff
@@ -126,6 +126,13 @@ cdef int ssl_SSL_ERROR_WANT_READ = ssl.SSL_ERROR_WANT_READ
 cdef int ssl_SSL_ERROR_WANT_WRITE = ssl.SSL_ERROR_WANT_WRITE
 cdef int ssl_SSL_ERROR_SYSCALL = ssl.SSL_ERROR_SYSCALL
 
+cdef FATAL_ERROR_IGNORE = (
```

Comment:
Can we rename this to FATAL_SSL_ERROR_IGNORE

Reply:
Makes sense! Will do, thx.